refactor some textsmells (#61)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

Diffusion models achieve state-of-the-art synthesis results on image data by decomposing the image formation process into sequential applications of denoising autoencoders. Due to its work directly in pixel space, they are very computational expensive. To lower the computational costs, the Stable Diffusion model (v1.5) is applied in a latent space of powerful pretrained autoencoders. This works without losing quality and flexibility. \cite{Rombach_2022_CVPR}

As the second image generative model we use \textbf{OpenDalle} from dataautogpt3 (\url{https://huggingface.co/dataautogpt3/OpenDalle}). OpenDalle, as it is based on the Dall-E model, works by modelling text and image tokens as a single data stream. This is achieved by training a transformer to autoregressively model those tokens into the resulting stream.

> This is done in two steps: Firstly, The images get compressed in a $32 \times 32$ grid of image tokens to reduce the context size of the transformer. Secondly, BPE-encoded text tokens are concatenated with the image tokens to model a joint distribution over text and image tokens with the training of an autoregressive transformer.

Reviewer: If we remove the newline here, I would also remove the line-break afterwards, so we merge these two paragraphs

Author: dalle and midjourney are not the same model why would you merge them

Reviewer: No I mean merging line 66 and line 67 to one paragraph.

Author: they are one paragraph i think no?

Reviewer: Yeah, but they still contain a line-break. This does not change the resulting report, so we can also just leave it as is.

\subsection{Detector models}

> \textbf{Winston AI} is a commercial AI text detector. They describe two primary methods for checking if a text is written by an AI. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness, another is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}

Reviewer 2: ```suggestion
\textbf{Winston AI} is a commercial AI text detector. They describe two primary methods for checking if a text is written by an AI model. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness, another is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}
```

> \textbf{Winston AI} is a commercial AI text detector. They describe two primary methods for checking if a text is written by an AI model. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness, another is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}

Reviewer 3: ```suggestion
\textbf{Winston AI} is a commercial AI text detector, which consists of two primary methods for checking if a text is written by an AI model. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness, another is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}
```

\section{Method}
To test and evaluate the effect of different data augmentation techniques on the detection capabilities of fake image or text detectors, we created a pipeline of three components: The first component is a generator, which is a model capable of generating fake images or texts. The second component is a processor which augments the data through a specified method, after given a generated image or text. The last component is a detector, which suggests an input being either real or fake. This way we can compare it to the ground truth, to evaluate the capabilities of the detector. By comparing the results with augmented artefacts and with the generated ones without augmentation, we can evaluate the effectiveness of different AI detector-tricking techniques.

> We used some of the aforementioned models for generating and detecting texts and images. For generating we used Stable Diffusion, OpenDalle, GPT2 and Falcon-RW-1B. For detecting we used RoBERTa Base OpenAI Detector, RADAR-Vicuna-7B, AIorNOT and ai-image-det-resnet18. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. We used gaussian noice, because it can occure due to natural sources and it is the most common noise that affects images naturally. \cite{mohammed2016study}

Reviewer 2: ```suggestion
We used some of the aforementioned models for generating and detecting texts and images. For generating we used Stable Diffusion, OpenDalle, GPT2 and Falcon-RW-1B. The detections were done with the RoBERTa Base OpenAI Detector, RADAR-Vicuna-7B, AIorNOT and ai-image-det-resnet18. As for image processors, one was already given in the description of the "InformatiCup2024" \cite{InformatiCup}, the image noising using the Gaussian noise. Image noise is the random variation of brightness or color information in images produced by the sensor and circuitry of a scanner or digital camera. We used gaussian noice, because it can occure due to natural sources and it is the most common noise that affects images naturally. \cite{mohammed2016study}
```

There are several types of noise that can affect images. Some of these noise models are: Gaussian noise, White noise, Fractal noise, Salt \& Pepper noise, Periodic noise, Quantization noise, Speckle noise, Poisson noise, Poisson-Gaussian noise, Structured noise, Gamma noise and Rayleigh noise \cite{gonzalez2009digital}. The three common types of image noise are: Gaussian noise, Salt \& Pepper noise, and Speckle noise \cite{al2010comparative}. That is why we used those three, as well as the Poisson noise.

> As for text processors, we evaluated three different methods of augmenting generated texts. The first approach was to replace every white-space in a given text by two white-spaces. The second approach was to add some typos to the texts. This has been implemented based on the nlpaug library, which is capable of simulating certain typos as if they were made on a keyboard by a human. The typos include mistakes, like hitting a wrong key, which is close to the right one. This results in typos often created by humans. The last approach is a translator, which has been used to translate the given text into a certain language (in this case french) and re-translate it into english. As translators are not deterministic but aim to keep the semantic meaning, the resulting text should be semantically the same, but might have a different formulation.

Reviewer: This means the Name of the Language, which is French and therefore should stay uppercase.
```suggestion
As for text processors, we evaluated three different methods of augmenting generated texts. The first approach was to replace every white-space in a given text by two white-spaces. The second approach was to add some typos to the texts. This has been implemented based on the nlpaug library, which is capable of simulating certain typos as if they were made on a keyboard by a human. The typos include mistakes, like hitting a wrong key, which is close to the right one. This results in typos often created by humans. The last approach is a translator, which has been used to translate the given text into a certain language (in this case French) and re-translate it into english. As translators are not deterministic but aim to keep the semantic meaning, the resulting text should be semantically the same, but might have a different formulation.
```

Author: shouldnt english not also be uppercase then?

Reviewer: > shouldnt english not also be uppercase then?

Yes, you're right.

Author: changed both to uppercase now
